start.bash

```diff
@@ -74,14 +98,16 @@ if [[ "$argo_server_alive" == "yes" ]]; then
 else
     $(dirname $0)/sqlflow/scripts/travis/start_argo.sh
 fi
+wait_until "argo" "is_resource_alive argo service/argo-server" "yes"
 
-echo "Strat Kubernetes Dashboard..."
+echo "Strat Kubernetes Dashboard ..."
 dashboard_alive=$(is_resource_alive "kubernetes-dashboard" "service/kubernetes-dashboard")
 if [[ "$dashboard_alive" == "yes" ]]; then
     echo "Already in running."
 else
     nohup minikube dashboard &
 fi
+wait_until "Kubernetes Dashboard" "is_resource_alive kubernetes-dashboard service/kubernetes-dashboard" "yes"
```

Comment:
Using a timeout threshold to avoid endless loop
``` bash
for _ in {1..30}; do
    POD_READY=$(kubectl get pod ${name} -o jsonpath='{.status.containerStatuses[0].ready}')
    if [[ "${POD_READY}" == "true" ]]; then
       break
    else
       sleep 3
done
```

Reply:
done